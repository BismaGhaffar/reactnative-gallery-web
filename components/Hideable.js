import styled from 'styled-components'

const Hideable = styled.span`
  @media (max-width: 481px) {
    display: ${props => (props.xs ? 'none' : 'block')};
  }
  @media (min-width: 481px) {
    display: ${props => (props.md ? 'none' : 'block')};
  }
`

export default Hideable
