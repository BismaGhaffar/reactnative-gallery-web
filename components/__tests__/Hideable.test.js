import React from 'react'
import renderer from 'react-test-renderer'
import Hideable from '../Hideable'

it('Hideable can be created', () => {
  const comp = renderer.create(<Hideable />)
  expect(comp).toBeDefined()
})

it('<Hideable /> toMatchSnapshot', () => {
  const tree = renderer.create(<Hideable />).toJSON()
  expect(tree).toMatchSnapshot()
})
