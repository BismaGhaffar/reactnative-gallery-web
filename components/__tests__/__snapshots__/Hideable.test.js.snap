// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hideable /> toMatchSnapshot 1`] = `
<span
  className="Hideable-lb4mgy-0 chPphA"
/>
`;
